crmd/lrm.c

```diff
@@ -2077,6 +2077,19 @@ do_lrm_rsc_op(lrm_state_t * lrm_state, lrmd_rsc_info_t * rsc, const char *operat
         }
     }
 
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc->id)) {
```

Comment:
It's because the remote ra is actually internal and defined by developers not by users. It does not fit into the concept of changing the resource agent code and therefore updating the version.